### PR TITLE
Fix typos in config.md code examples

### PR DESCRIPTION
**Pull request by eprikazc**

### CODE REVIEW

**Review by deliseev — APPROVED** [review]

File: docs/config.md [hunk 1]
```diff
@@ -139,7 +139,8 @@ SECRET_KEY = config('SECRET_KEY', cast=Secret)
 
 DATABASE_URL = config('DATABASE_URL', cast=URL)
 if TESTING:
-    DATABASE_URL = DATABASE_URL.replace(database='test_' + DATABASE_URL.database)
+    db_name = DATABASE_URL.path.lstrip('/')
+    DATABASE_URL = DATABASE_URL.replace(path='/test_' + db_name)
```
Comment (Kludex) [inline docs/config.md]: I guess the real intentions here were those: https://github.com/encode/starlette/pull/774
```suggestion
    DATABASE_URL = DATABASE_URL.replace(database='test_' + DATABASE_URL.database)
```

Comment (Kludex) [reply docs/config.md]: This will be fixed on https://github.com/encode/starlette/pull/774/

File: docs/config.md [hunk 2]
```diff
@@ -139,6 +139,7 @@ SECRET_KEY = config('SECRET_KEY', cast=Secret)
 
 DATABASE_URL = config('DATABASE_URL', cast=URL)
 if TESTING:
+    db_name = DATABASE_URL.path.lstrip('/')
```
Comment (Kludex) [inline docs/config.md]:
```suggestion
```

**Review by Kludex — APPROVED** [review]
Thanks @eprikazc ! :tada:

**Pull request merged**